doc/contributing/style_guides/markdown_usage_style.md

```diff
@@ -37,277 +39,131 @@ No style guide is perfect.
-Code fragments are encouraged.
-
-_This section is primarily for software engineers, but it is expected that the code fragments are used by verification engineers._
-
-```
-
-# Programmers Guide
 
-```
+## Filename
 
-One important thing here is to show the order of initialization that has been tested in the verification environment.
-In most cases other orders will work, and may be needed by the software environment, but it can be helpful in tracking down bugs for the validated sequence to be described!
+The Markdown file names should be in snake case and should use the `.md` file extension.
 
-```
-## Initialization
-```
-```c
+## Extensions
 
- if (...) {
-   a = ...
- }
-```
+One must only use [CommonMark][] with the exception of the following extensions:
+- [GFM Tables](https://github.github.com/gfm/#tables-extension-)
+- [GFM List Items](https://github.github.com/gfm/#task-list-items-extension-)
+- [Inline maths and maths blocks](#maths)
 
-Other sections cover different use cases and example code fragments.
+## Line length and Whitespace
 
-```
+Files must have exactly one sentence per line, with no line breaks in the middle of a sentence.
+This way change reviews will highlight only those sentences which are modified.
+Though the long line lengths make the files slightly less convenient to read from the command-line, this greatly simplifies the review process.
+When reviewing Markdown changes, every altered sentence will be included in its entirety in the file diff.
 
-## Use case A (eg Transmission)
+Markdown files must contain no trailing white space.
+As a result, two or more spaces must not be used for [hard line breaks][].
+Instead, a double new line is recommended.
+[Backslash hard breaks][] can also be used.
 
-## Use case B (eg Reception)
+## HTML
 
-```
+Do not use inline HTML within markdown documents.
+Although it offers a lot of flexibility, this flexibility is hard to handle.
+- It is possible to break the website
+- Markdown renderers often support different subsets of HTML.
+- The use of HTML restricts the ease at which documentation could be converted to other forms in the future, such as PDF.
 
-It is important to include a discussion of error conditions.
+Reaching for HTML may suggest that:
+- You need to split the concepts into more smaller parts that can be expressed in markdown syntax.
+- The concept would be better explained in an svg diagram.
 
-```
-## Error conditions
+*The one exception to this is [comments](#comments).*
 
-```
+## Links
 
-Also comment on anything special about interrupts, potentially including the priority order for servicing interrupts.
+The [GFM autolinks extention][] is not used so links should be flanked by `<` and `>` characters.
 
+One must not use [shortcut reference link][]s, but instead use [collapsed reference link][]s.
+This is because it can be ambiguious whether [shortcut reference link][] are meant to be links or not.
```

Comment:
Sorry to hear this, the shortcut links are much nicer to read.

Maybe an inline example would be useful here so you don't have to click through to what collapsed and shortcut reference links are.

Reply:
Examples added